frame/dex/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+// This file is part of Substrate.
```

Comment:
didn't we deprecated custom RPC?

Reply:
Yeah, anything new should be behind `state_call`

Reply:
Thanks for pointing that out @xlc, have removed the RPC and tested out that `state_call` works as expected.